DotNet.Bundle/build/DotNet.Bundle.targets

```diff
@@ -33,15 +32,26 @@
       <CFBundleIconFile Condition="'$(CFBundleIconFile)' == '' AND '$(TargetName)' != ''">$(TargetName).icns</CFBundleIconFile>
       <CFBundleIconFile Condition="'$(CFBundleIconFile)' == '' AND '$(TargetName)' == ''">none.icns</CFBundleIconFile>
 
-      <CFBundleShortVersionString Condition="'$(CFBundleVersion)' == ''">$(CFBundleVersion)</CFBundleShortVersionString>
+      <CFBundleShortVersionString Condition="'$(CFBundleShortVersionString)' == ''">$(CFBundleVersion)</CFBundleShortVersionString>
+
+      <CFBundleInfoDictionaryVersion Condition="'$(CFBundleInfoDictionaryVersion)' == ''">6.0</CFBundleInfoDictionaryVersion>
+
+      <CFBundleURLTypesValues>@(CFBundleURLTypes)</CFBundleURLTypesValues>
 
       <NSPrincipalClass Condition="'$(NSPrincipalClass)' == ''">NSApplication</NSPrincipalClass>
       
-      <NSHighResolutionCapable Condition="'$(NSHighResolutionCapable)' == ''" >true</NSHighResolutionCapable>
-      
-    </PropertyGroup>
+      <CFBundleURLTypesValues>@(CFBundleURLTypes)</CFBundleURLTypesValues>
     
+      <LSEnvironmentValues>@(LSEnvironemnt)</LSEnvironmentValues>

```

Comment:
typo

Reply:
Fixed. Had to slightly change how we handle ItemGroups (ITaskItem)